cluster/gce/trusty/master.yaml

```diff
@@ -213,6 +213,7 @@ script
 	set -o errexit
 	set -o nounset
 
+	. /etc/kube-configure-helper.sh
```

Comment:
On the master branch, there is a sleep 60 before this line w/ a TODO. Should that be backported?